src/shared/Microsoft.AzureRepos/AzureReposHostProvider.cs

```diff
@@ -176,6 +172,29 @@ private async Task<ICredential> GenerateCredentialAsync(InputArguments input)
             return new GitCredential(result.AccountUpn, pat);
         }
 
+        private string[] GetScopes()
+        {
+            string @namespace;
+            if (_context.Settings.TryGetSetting(
+                    AzureDevOpsConstants.EnvironmentVariables.GcmDevOpsScope,
+                    Constants.GitConfiguration.Credential.SectionName,
+                    AzureDevOpsConstants.GitConfiguration.Credential.DevOpsScope,
+                    out @namespace) 
+                ||
+                _context.Settings.TryGetSetting(
+                    AzureDevOpsConstants.EnvironmentVariables.GcmVstsScope,
+                    Constants.GitConfiguration.Credential.SectionName,
+                    AzureDevOpsConstants.GitConfiguration.Credential.VstsScope,
+                    out @namespace) )
+            {
+                return @namespace.Split(new char[]{' ', ',', ';', '|'}, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
```suggestion
                // Also split on whitespace in case users set values like "a, b"
                return scopes.Split(new char[]{' ', ','}, StringSplitOptions.RemoveEmptyEntries);
```